From 4d131a4dbeb4aee6c64a6981623d9efb393674b9 Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca@z3ntu.xyz>
Date: Sat, 17 Apr 2021 18:08:32 +0200
Subject: [PATCH 1/2] Prefix MALIIT variables with MALIIT_PREFIX

Otherwise the files will land somewhere where we don't want them during
installation.
---
 config.pri            | 4 ++--
 src/plugin/plugin.pro | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/config.pri b/config.pri
index 9593c6c5..5001ac7b 100644
--- a/config.pri
+++ b/config.pri
@@ -50,7 +50,7 @@ enable-opengl {
 
 LOMIRI_KEYBOARD_PACKAGENAME = lomiri-keyboard
 LOMIRI_KEYBOARD_VERSION = $$system(cat $$PWD/VERSION)
-LOMIRI_KEYBOARD_DATA_DIR = "$${MALIIT_PLUGINS_DATA_DIR}/lomiri-keyboard"
+LOMIRI_KEYBOARD_DATA_DIR = "$${MALIIT_PREFIX}/$${MALIIT_PLUGINS_DATA_DIR}/lomiri-keyboard"
 # FIXME: make it configurable while still has a sensible default
 # on Debian-like multiarch LIBDIR.
 LOMIRI_KEYBOARD_LIB_DIR = "$${PREFIX}/lib/lomiri-keyboard"
@@ -62,7 +62,7 @@ PINYIN_DATA_DIR = "$$system(pkg-config --variable pkgdatadir libpinyin)/data"
 
 DEFINES += LOMIRI_KEYBOARD_DATA_DIR=\\\"$${LOMIRI_KEYBOARD_DATA_DIR}\\\"
 DEFINES += LOMIRI_KEYBOARD_PLUGIN_DIR=\\\"$${LOMIRI_KEYBOARD_PLUGIN_DIR}\\\"
-DEFINES += MALIIT_PLUGINS_DATA_DIR=\\\"$${MALIIT_PLUGINS_DATA_DIR}\\\"
+DEFINES += MALIIT_PLUGINS_DATA_DIR=\\\"$${MALIIT_PREFIX}/$${MALIIT_PLUGINS_DATA_DIR}\\\"
 DEFINES += PINYIN_DATA_DIR=\\\"$${PINYIN_DATA_DIR}\\\"
 
 unix {
diff --git a/src/plugin/plugin.pro b/src/plugin/plugin.pro
index c76d38b0..33552d6a 100644
--- a/src/plugin/plugin.pro
+++ b/src/plugin/plugin.pro
@@ -40,7 +40,7 @@ SOURCES += \
     keyboardsettings.cpp \
 #    updatenotifier.cpp \
 
-target.path += $${MALIIT_PLUGINS_DIR}
+target.path += $${MALIIT_PREFIX}/$${MALIIT_PLUGINS_DIR}
 INSTALLS += target
 
 include(../word-prediction.pri)
-- 
2.33.0

